ui/lib/apps/UserProfile/translations/zh.yaml

```diff
@@ -40,8 +40,8 @@ user_profile:
       copied: 已复制
   version:
     title: 版本信息
-    internal_version: TiDB Dashboard 内部版本号
-    build_git_hash: TiDB Dashboard 编译 Git Hash
-    build_time: TiDB Dashboard 编译时间
-    standalone: TiDB Dashboard 运行于独立模式
-    pd_version: PD 版本号
+    internal_version: '{{distro.tidb}} Dashboard 内部版本号'
+    build_git_hash: '{{distro.tidb}} Dashboard 编译 Git Hash'
+    build_time: '{{distro.tidb}} Dashboard 编译时间'
+    standalone: '{{distro.tidb}} Dashboard 运行于独立模式'
+    pd_version: '{{distro.pd}} 版本号'
```

Comment:
It seems some have single quotes, some don't have?

Reply:
Yeah, the parentheses after the colon will be recognized as hash map definition. So we need these quotes.